lib/class-affiliates-import-process.php

```diff
@@ -375,6 +375,7 @@ class_exists( 'Affiliates_Registration' ) &&
 
 		$user_id = Affiliates_Registration::create_affiliate( $userdata );
 		if ( !is_wp_error( $user_id ) ) {
+			Affiliates_Registration::new_user_notification( $user_id, $userdata['password'] );
```

Comment:
This must take the option into account http://docs.itthinx.com/document/affiliates-import/ - "Send new users their password by email"

Reply:
But of course you're right